codesign/test/codesign_test.dart

```diff
@@ -0,0 +1,12 @@
+// Copyright 2022 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:codesign/codesign.dart';
+import 'package:test/test.dart';
+
+void main() {
+  test('calculate', () {
```

Comment:
You'll need to add an entry in [tests.yaml](https://cs.opensource.google/flutter/cocoon/+/main:tests.yaml) so this runs on infra